tests/playwright/src/specs/extension-installation.spec.ts

```diff
@@ -50,8 +50,8 @@ let ociImageUrl: string;
 let navigationBar: NavigationBar;
 
 async function _startup(extensionName: string): Promise<void> {
-  pdRunner = await PodmanDesktopRunner.getInstance();
-  page = await pdRunner.start();
+  pdRunner = await Runner.getInstance();
```

Comment:
Why is it called `pdRunner` here?

Reply:
Because that is local to that test, it's not from the fixture, that test suite calls has a very weird flow where the beforeAll/afterAll hooks are called as part of some function, I didn't make any changes to that test file that I didn't absolutely need, which is why it's not using fixtures.